guide/onboarding/creating-a-new-wallet.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: guide
+title: Creating a new wallet
+description: Steps to take when creating a new wallet
+parent: Onboarding
+nav_order: 2
+permalink: /guide/onboarding/creating-a-new-wallet/
+main_classes: -no-top-padding
+---
+
+# Creating a new wallet
+
+First, bitcoin [wallets]({{ 'guide/glossary/#wallet' | relative_url }}) are not analogous to their namesakes. Physical wallets are places to store physical currency, credit cards, IDs, and so on. Bitcoin wallets don't store bitcoin. However, the [self custodial]({{ 'guide/glossary/#self-custody' | relative_url }})  nature of bitcoin wallets gives users full control of what they do with their bitcoin.
+
+Instead, a bitcoin wallet contains the "private keys" to your bitcoin. The bitcoin is located on the Bitcoin network that contains a public ledger similar in many ways to a spreadsheet with a record of everyone's transactions and balances. Private keys let you access bitcoin associated with those keys. A more suitable analogy for a bitcoin wallet would be a keychain.
+
+ Like regular keys on a keychain, bitcoin private keys can be copied. For example, if you gave someone a copy of your house keys, they can unlock the door to your home. Likewise, if someone has a copy of your private key, they can access your bitcoin. This makes [private key management]({{ '/guide/private-key-management/introduction' | relative_url }}) one of the most important considerations of any bitcoin owner.
+
+ Most bitcoin wallets will automatically generate users private keys, it is usually not something a user controls. We will look at practical design solutions for guiding users through this process in the next section. For now, this page explains how we can help get a user comfortable with the idea of a self-custodial wallet.
+
+There are a few different types of wallets that let you send, receive, store and manage bitcoin. For this page, we focus on regular single-key wallets which are the ones most commonly used.
```

Comment:
```suggestion
A few different types of wallets let you send, receive, store, and manage bitcoin. For this page, we focus on regular single-key wallets, which are the ones most commonly used.
```

Reply:
Might be worth adding that it mostly focuses on mobile wallets and not desktop / web. 

Also add "...we focus on regular single-key self-custody wallets,..."